frontend/src/css/Generic.styl

```diff
@@ -23,10 +23,11 @@ a
   &:disabled
     cursor not-allowed
 
-div.tox-tinymce
+// INFO - CH - 2025-02-19 - Set minimum lines in HugeRTE textarea to 4
+div.tox-hugerte
```

Comment:
Value of one line is 37,5px ? Maybe add this information in the comment.

Reply:
Not exactly, it's more of a "look alike" design. If you add a line in the comment textarea, its height will increase at the first one, not after the fourth